storage/backend/ceph/util.go

```diff
@@ -13,7 +14,13 @@ func (c *Driver) MountPath(do storage.DriverOptions) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return filepath.Join(c.mountpath, do.Volume.Params["pool"], volName), nil
+
+	poolName, ok := do.Volume.Params["pool"].(string)
+	if !ok {
+		return "", errored.Errorf("Expected string type for driver.pool")
```

Comment:
Yeah. Please take a look now. I have added an another abstraction layer. Now, there is a generic `Get` for both nfs, ceph. And, I did not add any thing in `errors.go`; I feel its not required for the current design.  Let me know what you think.